va_explorer/va_analytics/dash_apps/va_dashboard.py

```diff
@@ -1410,8 +1777,10 @@ def trend_plot(va_data, timeframe, group_period, filter_dict=None, factor="All",
             figure = plotting.va_trend_plot(
                 plot_data, group_period, factor, title=plot_title, search_term_ids=search_term_ids, height=560
             )
+            # print("TIMEEEEEEEEEEEEEEEEEEE")
+            # print(time.time() - START_TIME)
+    print("trend plot Callback", time.time() - START_TIME)
     return dcc.Graph(id="trend_plot", figure=figure, config=LOOKUP["chart_config"])
 
-
 # uncomment this if running as Dash app (as opposed to DjangoDash app)
-# app.run_server(debug= True)
+# app.run_server(debug= True)
```

Comment:
delete...? I notice you added line 1786, but line 1785 was already there. So what was 1785 referring to before?

delete both imo

Reply:
This was here before I started editing. Not sure why it looks like I edited it. Will leave it in in case Brendan wants to continue using it.